### PR TITLE
Test: add a sample test case for pkg manager

### PR DESCRIPTION
**Pull request by SamYuan1990**
part of #434 

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
hi @rootfs 
for no bcc tag
```
Manager
/home/runner/work/kepler/kepler/pkg/manager/manager_test.go:8
  [It] Should work properly
  /home/runner/work/kepler/kepler/pkg/manager/manager_test.go:10

  Unexpected error:
      <*errors.errorString | 0xc0002f0190>: {
          s: "failed to attach bpf assets: no bcc build tag",
      }
      failed to attach bpf assets: no bcc build tag
  occurred
```
for bcc test
```
Manager
/home/pkg/manager/manager_test.go:8
  [It] Should work properly
  /home/pkg/manager/manager_test.go:10

  Unexpected error:
      <*errors.errorString | 0xc0000400e0>: {
          s: "failed to attach bpf assets: failed to attach the bpf program: <nil>",
      }
      failed to attach bpf assets: failed to attach the bpf program: <nil>
  occurred
  In [It] at: /home/pkg/manager/manager_test.go:14
------------------------------
```

File: pkg/manager/manager_test.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package manager
```
Comment (rootfs) [inline pkg/manager/manager_test.go:1]: add bcc build tag to avoid getting tested on non-bcc platform

Comment (rootfs) [reply pkg/manager/manager_test.go:1]: see https://github.com/sustainable-computing-io/kepler/pull/441/files#diff-d765c874d837c9e7b84161e25b51a9be7854289d92c4113f5dc96c76bb017d8fR1

Comment (SamYuan1990) [reply pkg/manager/manager_test.go:1]: https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L51
here is the rc, as bcc tag will added only for ? I suppose centos yum? but for github action, os is ubuntu.

Comment (rootfs) [reply pkg/manager/manager_test.go:1]: the tag is used by `go test` here https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L145

**Comment by SamYuan1990** [comment]
```
=== RUN   TestManager
Running Suite: Manager Suite - /home/runner/work/kepler/kepler/pkg/manager
==========================================================================
Random Seed: 1670068146

Will run 1 of 1 specs
•

Ran 1 of 1 Specs in 0.002 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestManager (0.01s)
PASS
coverage: 55.6% of statements
ok  	github.com/sustainable-computing-io/kepler/pkg/manager	0.064s
```
https://github.com/sustainable-computing-io/kepler/actions/runs/3608369775/jobs/6080934611#step:6:323

**Review by rootfs — APPROVED** [review]

**Pull request merged**